hangman/README.md

```diff
@@ -1,3 +1,6 @@
-8k students only:
+# Hangman
+
+**TODO: Add description**
+
+The strtegy I used here is :permanent and the reason is that the server need running even in the case where the application exits for some reason and then restarting the game to run
```

Comment:
# -2

I think this should be transient, because if the server crashes, you've lost the game state anyway.